### ISSUE TITLE
No plugins handle this kind of module: Custom("asset")

### ISSUE THREAD

**Issue by houd1ni** [farm-fe/farm#1128]
### Steps to reproduce

Just a plain project that does

```html

<script setup lang="ts">
import handsupPic from '../assets/hands-up.png'

```

### What is expected?

png resolves to url

### What is actually happening?

![image](https://github.com/farm-fe/farm/assets/7501201/fbe3380e-27b5-4255-a3f8-ea607a97660d)
and many more

### System Info

```Shell
Win10x64 Pro latest, Intel 14600kf, 32Gb ram, almost empty.
```

### Any additional comments?

Struggled for several hours. All docs tell that it's built in. No plugins or settings related to images or assets help.

**Comment by callqh** [comment]
Can you take a look at your farm.config configuration and see if you have used any Vite plugins?

**Comment by callqh** [comment]
We have fixed this issue,please pay attention to #1127
